llpc/util/llpcCacheAccessor.cpp

```diff
@@ -85,8 +85,11 @@ void CacheAccessor::lookUpInCaches(const MetroHash::Hash &hash) {
   memcpy(&hashId.bytes, &hash.bytes, sizeof(hash));
 
   Result cacheResult = Result::Unsupported;
-  if (getInternalCache())
+  if (getInternalCache()) {
     cacheResult = lookUpInCache(getInternalCache(), !getApplicationCache(), hashId);
+    if (cacheResult == Result::Success)
+      m_internalICacheHit = true;
```

Comment:
When I first wrote this code, I was using ICache in the function names to try to differential it from the shader cache.  I was asked to change it to just "Cache".  That is why we have `getApplicationCache` instead of `getApplicationICache`.

You might be asked to change this.

Reply:
Fixed.